scripts/ci/wait_for_driver_to_be_ready.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+set +o pipefail
+
+driver_is_ready=false
+amount_of_seconds_that_driver_pods_are_running=0
+amount_of_seconds_that_driver_pods_should_be_running=10
+while [ "$(kubectl get pod -A -l csi | grep controller | wc -l)" -eq 0 ]; do
+  echo "The controller is not deployed"
+  sleep 1
+done
+while [ $driver_is_ready == "false" ]; do
+  if [ "$(kubectl get pod -A -l csi | grep -iv running | grep -iv name | wc -l)" -eq 0 ]; then
+    ((amount_of_seconds_that_driver_pods_are_running++))
+    if [ $amount_of_seconds_that_driver_pods_are_running -eq $amount_of_seconds_that_driver_pods_should_be_running ]; then
+      driver_is_ready=true
+    fi
+  else
+    amount_of_seconds_that_driver_pods_are_running=0
```

Comment:
so if the driver can't run, this script will be stuck forever?

Reply:
where is it defined? isn't it too much?

Reply:
not more than five minutes

Reply:
so could you just change it to [5](https://github.com/IBM/ibm-block-csi-driver/blob/2ba9b3a454da6f8f6ca65ba8b56d5d33e36e1bc6/.github/workflows/driver_ci.yaml#L147)?